src/main/java/reactor/pool/PoolBuilder.java

```diff
@@ -251,50 +253,33 @@
     }
 
     /**
-     * Let the {@link Pool} allocate between {@code min} and {@code max} resources.
+     * Replace the {@link AllocationStrategy} with one that lets the {@link Pool} allocate between {@code min} and {@code max} resources.
      * When acquiring and there is no available resource, the pool should strive to warm up enough resources to reach
      * {@code min} live resources before serving the acquire with (one of) the newly created resource(s).
      * At the same time it MUST NOT allocate any resource if that would bring the number of live resources
      * over the {@code max}, rejecting further allocations until some resources have been {@link PooledRef#release() released}.
      *
      * @param min the minimum number of live resources to keep in the pool (can be best effort)
-     * @param max the maximum number of live resources to keep in the pool
+     * @param max the maximum number of live resources to keep in the pool. use {@link Integer#MAX_VALUE} when you only need a
+     * minimum and no upper bound
      * @return this {@link Pool} builder
+     * @see #sizeUnbounded()
+     * @see #allocationStrategy(AllocationStrategy)
      */
     public PoolBuilder<T, CONF> sizeBetween(int min, int max) {
         return allocationStrategy(new AllocationStrategies.SizeBasedAllocationStrategy(min, max));
     }
 
     /**
-     * When acquiring and there is no available resource, the pool should strive to warm up enough resources to reach
```

Comment:
No depreciation, even for a little while?

Reply:
this is 0.0.1 M3-to-RC1 transition, there won't be another chance to remove this API post M1... I could probably put the deprecation in snapshots for a while, but the next "release" won't have the methods anyway. @mp911de any opinion?

Reply:
No need from our side. We need to migrate to the proper methods anyway. With this close communication, that's easy to achieve for us.